.travis.yml

```diff
@@ -5,6 +5,7 @@ rvm:
 - 2.1.5
 - 2.2.0
 - 2.3.0
+- jruby-9.1.5.0
```

Comment:
@vishrutshah @veronicagg I tried adding jruby-head and that does not work at present due to different issue. It can be seen here: https://github.com/travis-ci/travis-ci/issues/6085